[2024-10-03T05:56:38.601+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T05:56:38.661+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.staging.dellstore_db.extract.orders scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T05:56:38.681+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.staging.dellstore_db.extract.orders scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T05:56:38.681+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-03T05:56:38.742+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): staging.dellstore_db.extract.orders> on 2024-10-02 00:00:00+00:00
[2024-10-03T05:56:38.785+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T05:56:38.798+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'staging.dellstore_db.extract.orders', 'scheduled__2024-10-02T00:00:00+00:00', '--job-id', '13514', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmp_k15jjt9']
[2024-10-03T05:56:38.824+0000] {standard_task_runner.py:105} INFO - Job 13514: Subtask staging.dellstore_db.extract.orders
[2024-10-03T05:56:38.813+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=544) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-03T05:56:38.834+0000] {clientserver.py:505} INFO - Error while sending or receiving.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer
[2024-10-03T05:56:38.839+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T05:56:38.854+0000] {standard_task_runner.py:72} INFO - Started process 1421 to run task
[2024-10-03T05:56:38.854+0000] {java_gateway.py:1052} INFO - Exception while sending command.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 506, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending
[2024-10-03T05:56:38.883+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T05:56:39.064+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.staging.dellstore_db.extract.orders scheduled__2024-10-02T00:00:00+00:00 [running]> on host b9e3ad173054
[2024-10-03T05:56:40.628+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='staging.dellstore_db.extract.orders' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T00:00:00+00:00'
[2024-10-03T05:56:40.629+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T05:56:40.810+0000] {base.py:84} INFO - Retrieving connection 'dellstore_db'
[2024-10-03T05:56:42.124+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-10-03T05:56:43.054+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T05:56:43.325+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T05:56:43.326+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=staging.dellstore_db.extract.orders, run_id=scheduled__2024-10-02T00:00:00+00:00, execution_date=20241002T000000, start_date=20241003T055638, end_date=20241003T055643
[2024-10-03T05:56:44.276+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T05:56:45.599+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T05:56:45.628+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
